### PR TITLE
Add sample filtering + fix 'label' and 'path' bug in classification params

### PR DESCRIPTION
**Pull request by bartcharbon**
- new vcf-decision-tree version
- added classification and filter step for samples
- removed inheritance filtering
- version bump
- fix 'label' and 'path' bug in classification params in the variant classification step
- remove unused decision tree from resources (decision_tree_pick.json)
- Default sample decision tree 

### Before merge:
- [x] Functionality works & meets specs
- [x] No issues running `bash test/pipeline_test.sh`
- [x] Code reviewed
- [ ] Documentation was updated

### After merge:
- [ ] Added feature/fix to draft release notes

## Notes by PR author
None.

### CODE REVIEW

File: resources/decision_tree_samples.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      "outcomeFalse": {
+        "nextNode": "exit_r"
+      },
+      "outcomeMissing": {
+        "nextNode": "depth"
+      }
+    },
+    "depth": {
+      "type": "BOOL",
+      "description": "Depth",
```
Comment (dennishendriksen) [inline resources/decision_tree_samples.json:42]: Depth -> Read depth

File: resources/decision_tree_samples.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      "outcomeFalse": {
+        "nextNode": "exit_r"
+      },
+      "outcomeMissing": {
+        "nextNode": "exit_k"
+      }
+    },
+    "genotype_quality": {
+      "type": "BOOL",
+      "description": "Genotype quality",
```
Comment (dennishendriksen) [inline resources/decision_tree_samples.json:60]: Genotype quality (GATK)

**Review by dennishendriksen — CHANGES_REQUESTED** [review]
```
FAILED test_snv_proband_trio, see <cut>/test/output/test_snv_proband_trio/.nxf.log
```

```
[~]$ zcat <cut>/test/output/test_snv_proband_trio/snv_proband_trio.vcf.gz | grep -vc "^#"
3
```
expected value is 2: https://github.com/molgenis/vip/blob/master/test/test.sh#L124

**Review by dennishendriksen — CHANGES_REQUESTED** [review]
Add an integration test case (or update an existing one) with sample filtering enabled

File: resources/decision_tree_samples.json (hunk 2 above)
Comment (bartcharbon) [reply resources/decision_tree_samples.json:60]: is gewoon een VCF spec veld toch? niet specifiek GATK?

Comment (dennishendriksen) [reply resources/decision_tree_samples.json:60]: you are right, my bad.

**Review by dennishendriksen — APPROVED** [review]

**Pull request merged**